recipes/miniaudio/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+from conans import ConanFile, tools
+import os
+import glob
+
+
+class MiniaudioConan(ConanFile):
+    name = "miniaudio"
+    description = "A single file audio playback and capture library."
+    topics = ("conan", "miniaudio", "header-only", "sound")
+    homepage = "https://github.com/mackron/miniaudio"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = ["Unlicense", "MIT-0"]
```

Comment:
```suggestion
    license = ["Unlicense", "MIT"]
```

Reply:
The license is "MIT no attribution". the SPDX identifier for that one is MIT-0, which is different from MIT.

Reply:
right